src/tests/ftest/deployment/critical_integration.py

```diff
@@ -155,15 +155,18 @@ def test_ras(self):
             self.log.info("Ranks to stop: %s", ranks_to_stop)
             # stop ranks and verify if they stopped
             dmg.system_stop(ranks=ranks_to_stop)
-            for rank in sub_list:
-                if (not self.server_managers[0].check_rank_state(rank, "stopped", 5)
-                        or self.server_managers[0].check_rank_state(rank, "excluded", 5)):
-                    self.fail("Rank {} failed to stop".format(rank))
+            check_stopped_ranks = self.server_managers[0].check_rank_state(sub_list,
+                                                                           ["stopped", "excluded"],
+                                                                           5)
```

Comment:
Just FYI, this is also supported per https://pep8.org/#indentation:
```suggestion
            check_stopped_ranks = self.server_managers[0].check_rank_state(
                sub_list, ["stopped", "excluded"], 5)
```